submarine-sdk/setup.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    LONG_DESCRIPTION = fh.read()
+
+setup(
+    name='pysubmarine',
```

Comment:
Is the name `pysubmrine` or `submarine-sdk` or `submarine-tracking`?
I think, `pysubmrine` may be another meaning, refers to the docking and development of submarine completely through python?
Please ask this question in the next community meeting and discuss it together. Thanks.

Reply:
OK, no problem